modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1743,11 +1743,6 @@
 			<description name="Replace parameter type long by ItemInfoReference in the getLayoutDisplayPageObjectProvider method" />
 			<weight>1</weight>
 		</check>
-		<check name="UpgradeJavaGetLayoutDisplayPageProviderCheck">
-			<category name="Upgrade" />
-			<description name="Replace getLayoutDisplayPageProvider by getLayoutDisplayPageProviderByClassName" />
-			<weight>2</weight>
-		</check>
```

Comment:
@alberttgomes this one had a weight of 2. This means that it depends on another check. Maybe this one shouldn't be migrated. Let's discuss it tomorrow. Michael did this one, ask him what It depends.

Reply:
That's ok! @nicolas-sss